File: MPChartExample/src/com/xxmassdeveloper/mpchartexample/fragments/SineCosineFragment.java
package com.xxmassdeveloper.mpchartexample.fragments;
import android.graphics.Typeface;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.github.mikephil.charting.charts.LineChart;
import com.github.mikephil.charting.components.Legend;
import com.github.mikephil.charting.components.XAxis;
import com.github.mikephil.charting.components.YAxis;
import com.xxmassdeveloper.mpchartexample.R;


public class SineCosineFragment extends SimpleFragment {

    public static Fragment newInstance() {
        return new SineCosineFragment();
    }

    private LineChart mChart;
    
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.frag_simple_line, container, false);
        
        mChart = (LineChart) v.findViewById(R.id.lineChart1);

        mChart.getDescription().setEnabled(false);

        mChart.setDrawGridBackground(false);
        
        mChart.setData(generateLineData());
        mChart.animateX(3000);
        
        Typeface tf = Typeface.createFromAsset(getActivity().getAssets(),"OpenSans-Light.ttf");
        
        Legend l = mChart.getLegend();
        l.setTypeface(tf);
        
        YAxis leftAxis = mChart.getAxisLeft();
        leftAxis.setTypeface(tf);
        leftAxis.setAxisMaximum(1.2f);
        leftAxis.setAxisMinimum(-1.2f);
        
        mChart.getAxisRight().setEnabled(false);
        
        XAxis xAxis = mChart.getXAxis();
        xAxis.setEnabled(false);
        
        return v;
    }
}

File: MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/fragments/SineCosineFragment.kt
package com.xxmassdeveloper.mpchartexample.fragments

import android.graphics.Typeface
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.mikephil.charting.charts.LineChart
import com.xxmassdeveloper.mpchartexample.R

class SineCosineFragment : SimpleFragment() {
    private var chart: LineChart? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val v = inflater.inflate(R.layout.frag_simple_line, container, false)
        chart = v.findViewById(R.id.lineChart1)
        chart?.description?.isEnabled = false
        chart?.setDrawGridBackground(false)
        chart?.data = generateLineData()
        chart?.animateX(3000)
        val tf = Typeface.createFromAsset(requireContext().assets, "OpenSans-Light.ttf")
        val l = chart?.legend
        l?.typeface = tf
        val leftAxis = chart?.axisLeft
        leftAxis?.typeface = tf
        leftAxis?.axisMaximum = 1.2f
        leftAxis?.axisMinimum = -1.2f
        chart?.axisRight?.isEnabled = false
        val xAxis = chart?.xAxis
        xAxis?.isEnabled = false
        return v
    }

    companion object {
        fun newInstance(): Fragment {
            return SineCosineFragment()
        }
    }
}